.editorconfig

```diff
@@ -0,0 +1,261 @@
+root = true
+
+[*]
+ij_continuation_indent_size = 8
+ij_formatter_off_tag = @formatter:off
+ij_formatter_on_tag = @formatter:on
+ij_formatter_tags_enabled = false
+ij_smart_tabs = false
+ij_visual_guides = none
+ij_wrap_on_typing = false
+
+[*.java]
```

Comment:
Do you actually need that long style configuration for Java?

Reply:
We used a template for that:) the more the better, as they say
I've shortened it a bit